hudi-utilities/src/main/java/org/apache/hudi/utilities/sources/helpers/KafkaOffsetGen.java

```diff
@@ -290,6 +292,7 @@ public OffsetRange[] getNextOffsetRanges(Option<String> lastCheckpointStr, long
       numEvents = sourceLimit;
     }
 
+    // TODO remove
```

Comment:
can you please create follow up tickets if you feel something needs to be worked up. if we have groomed tickets, we can ask someone from the community to take it up as well.